x-pack/plugins/security_solution/server/lib/detection_engine/rule_types/esql/utils/generate_alert_id.test.ts

```diff
@@ -142,15 +142,5 @@ describe('generateAlertId', () => {
       modifiedIdParams.completeRule.ruleParams.query = 'from packetbeat*';
       expect(id).toBe(generateAlertId(modifiedIdParams));
     });
-
-    it('creates id dependant on suppression terms', () => {
```

Comment:
We can/should still document this behavior with a unit test, similar to the one above: `it('creates id independent of suppression terms')`

Reply:
suppression terms are no longer part of this utility arguments. So, there is no need to add such tests.

Some of of there existing tests cover cases for aggr and non-aggr queries, which have different set of argument they depend on. That's why we have tests like these